app/views/workshops/show.html.haml

```diff
@@ -18,7 +18,9 @@
     .row
       .medium-12.columns
         %p.lead
-          Attend our workshops to learn programming in a safe and supportive environment at your own pace, or to share your knowledge and coach our students.
+          = t("workshops.lead")
+        %p.description
+          = sanitize(@workshop.description.html_safe)
```

Comment:
Almost there ... so html_safe allows HTML tags and Javascript and sanitize allows only HTML tags. so you can remove the html_safe and it does the same thing. We want to get rid of html_safe because it's very misleading.

Reply:
Thanks for clarifying this! Removed html_safe.